### ISSUE TITLE
Flickering when double clicking or selecting text

### ISSUE THREAD

**Issue by BrontoRed** [piroor/textlink#80]
## Short description
Flickering appears on some sites when double clicking or selecting text.

## Steps to reproduce

 1. Start Firefox with clean profile.
 2. Install Text Link.
 3. Open [URL_1](https://www.foxnews.com/sports/pga-tour-pro-matt-wallace-gets-heated-argument-caddie-tournament) or [URL_2](https://edition.cnn.com/travel/article/qatar-day-trip-zekreet-peninsula/index.html).
 4. Try to double-click on titles or select text in titles.

## Expected result
Text should be selected without any "effects".

## Actual result
Text is selected but with fast flickering of page elements.

## Environment

 * Platform (OS): Windows 10
 * Version of Firefox: 111
 * Version (or revision) of Text Link:  6.1.7

**Comment by piroor** [comment]
The commit 08e83c47abaa245cd8f7adb9efca379a132d43c8 should reduce this kind problems. Could you try the development build? https://github.com/piroor/textlink/actions/runs/4490086227

**Comment by BrontoRed** [comment]
Thanks! This one works for the most part on Firefox Developer edition 112.0b5, but with some exceptions: for example, not working [here](https://d3.ru/).

**Comment by piroor** [comment]
Sorry I have no idea to solve the problem completely - it is a side effect of the commit c7bbfcf5d71d1519d3c575be4faa7e063ae31a11 which was introduced to support URL-like text containing account name link on Twitter like https://twitter.com/piro_or/status/1603681790983811072

**Comment by BrontoRed** [comment]
Yes, I see that it's a  trade-off. Sometimes it's better to keep things simple.
Maybe make this detection optional and let user decide.

**Comment by piroor** [comment]
I've introduced more workaround with the commit bce7300, for cases like https://d3.ru/ . I hope the change should work effectively...

**Comment by BrontoRed** [comment]
Good try, but here is another [problem URL](https://www.ofnisystems.com/). Firefox Developer edition 112.0b6, Text Link: 6.1.8.741.

**Comment by BrontoRed** [comment]
So far so good! Thanks for a quick fix.

**Comment by BrontoRed** [comment]
Good try, but here is another [problem URL](https://www.ofnisystems.com/). Firefox Developer edition 112.0b6, Text Link: 6.1.8.741.

**Comment by piroor** [comment]
OK, the case should be covered with v6.1.9 with the commit 4cbb2b5.

**Comment by BrontoRed** [comment]
So far so good! Thanks for a quick fix.

**Comment by BrontoRed** [comment]
Please, check one more URL with this issue: https://www.calend.ru/holidays/hungary/
